Feature/settings

MurhafSousli opened the pull request:
closes #46

MurhafSousli committed:
About page

MurhafSousli committed:
License page

MurhafSousli committed:
Settings page (WIP)

MurhafSousli committed:
Add loading spinner for Auth and Billing effects

MurhafSousli committed:
Improve header navbar

MurhafSousli committed:
refactor

MurhafSousli committed:
Update settings

adrianq commented:
This is still a work in progress. @MurhafSousli to give a shout when ready!

MurhafSousli committed:
Notifcations: Add loading & confirmation dialogs for accept/reject users

MurhafSousli committed:
Remove 'serverHost' from user interface

MurhafSousli committed:
Refactor and fix liniting

MurhafSousli committed:
Improve main menu, use ion-button for menu items

MurhafSousli commented:
The commit [Notifcations: Add loading & confirmation dialogs for accept/reject users](https://github.com/EyeSeeTea/FIRE-WiFiCalling/pull/49/commits/e0245f6a6dbebac48b33f5a1c38004c0bf7ed1f7) in addition to that it also Adds and fixes translations for all pages (couldn't rename the commit 😕)

MurhafSousli committed:
Improve and cleanup styles

MurhafSousli committed:
refactor(Admin)

MurhafSousli committed:
Move auth state out to root module

fixes(auth state is undefined until user goes to login page)

 - initialize auth state  on app launch
 - remove redirect effect
 - remove auth guard

MurhafSousli committed:
refactor(Auth)

adrianq commented:
Improvements to the settings panel:
- [x] Non-logged user: Only server host is displayed and it is editable.
- [x] Logged user: Server host is displayed and it is editable. User name and password are displayed but non-editable. Whenever a user clicks on it, the user will be redirected to login page. Everytime a user is logged in we will store the new user and password if needed.
- [x] First time the application is started, server host will be empty. For development purposes, for now we will hardcode dev.est:5000

MurhafSousli committed:
Complete settings

MurhafSousli committed:
test travis build

MurhafSousli committed:
Improve dialogs

 - Create DialogService to create new dialogs easily
 - Improve dialogs styles

MurhafSousli committed:
Improve animations

 - Remove duplicate animations for AuthPage and TopUpPage tabs
 - Create shared animations file for common animations
 - Improve notifications' filter menu animation

MurhafSousli committed:
Create fire-tabs

 Use same tabs style for AuthPage and TopUpPage

MurhafSousli committed:
Remove back button text from header

MurhafSousli committed:
refactor global styles

MurhafSousli committed:
refactor(notifications)

 - Use Subject instead of input to mark notifications items as checked
 - Remove selected notifications when admin's tabs change
 - Improve filter icon for ios platform

MurhafSousli committed:
refactor(Settings)

 - Improve global styles for ios
 - Add disabled style for fire-input for disabled inputs in settings
 - Set "serverHost" as fire-http base url
 - fix wrong module name in settings.module

MurhafSousli committed:
TopUp

 - TopUp lazy page
 - TopUp store
 - TopUp Translation
 - Vouchers list
 - Add Voucher
 - Voucher detail

MurhafSousli committed:
small fix

 - Update translation
 - Refactor style

MurhafSousli committed:
tinyfix

MurhafSousli commented:
@adrianq PR is Ready

MurhafSousli committed:
Merge branch 'feature/menu' into feature/settings

MurhafSousli committed:
Remove wrong comment

adrianq committed:
Merge branch 'development' into feature/settings

adrianq committed:
fix wrong merge

adrianq committed:
Merge pull request #61 from EyeSeeTea/feature/topup

feature/topup

adrianq committed:
Merge branch 'feature/settings' of github.com:EyeSeeTea/FIRE-WiFiCalling into feature/settings

adrianq committed:
fix call page
